themes/default/content/docs/intro/concepts/resources/options/retainOnDelete.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "RetainOnDelete"
+meta_desc: The `retainOnDelete` resource option marks a resource to be retained during a delete operation.
+menu:
+  intro:
+    identifier: retainOnDelete
+    parent: options
+    weight: 13
+---
+
+The `retainOnDelete` resource option marks a resource to be retained. If a resource is retained then when
+Pulumi deletes or replaces the resource it will not call through to the resource provider's `Delete` method.
+As a result, the resource will not be deleted from the backing cloud provider, but will be removed from the
+Pulumi state.
+
+This option can be used for shared resources that other stacks might be making use of.
```

Comment:
Is this an example of when people want to do that? This sentence feels a bit hanging out of context and may be hard to grasp.